src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/target.py

```diff
@@ -275,11 +275,8 @@ def _target_is_ou_path(self, resolve_children=False):
                 ou_id=None,
                 excluded_paths=[],
             )
-        except ClientError as client_err:
-            no_target_found = (
-                client_err.response["Error"]["Code"] == "ParentNotFoundException"
-            )
-            if no_target_found and self.adf_deployment_maps_allow_empty_target:
+        except Exception:
+            if self.adf_deployment_maps_allow_empty_target:
```

Comment:
I don't think we should broaden the except type, as this would imply unrelated exceptions (think network or auth errors) would be hidden if `adf_deployment_maps_allow_empty_target` is set to true.

Could you please revert this change, then add support for another except that handles the error that it throws?